### PR TITLE
Validate the quota value to be a correct value

### PR DESCRIPTION
**Pull request by DeepDiver1975**
@jnfrmarks 

Please review @icewind1991 @blizzz @LukasReschke

### CODE REVIEW

**Comment by jnfrmarks** [comment]
@DeepDiver1975 

Can I get a bit more context?  What changed and what needs to be tested?  Thanks!

**Comment by LukasReschke** [comment]
```
12:17:57 1) Test_Helper_Storage::testGetStorageInfoIncludingExtStorage
12:17:57 Failed asserting that -22 matches expected 3.
12:17:57 
12:17:57 /var/jenkins/workspace/pull-request-analyser-ng-simple@3/label/SLAVE/tests/lib/helperstorage.php:129
```

**Comment by DeepDiver1975** [comment]
> Can I get a bit more context? What changed and what needs to be tested? Thanks!

I was your bug report - https://github.com/owncloud/enterprise/issues/269

File: lib/private/helper.php [hunk 1]
```diff
@@ -394,6 +394,9 @@ public static function phpFileSize($bytes) {
 	 */
 	public static function computerFileSize($str) {
 		$str = strtolower($str);
+		if (is_numeric($str)) {
```
Comment (LukasReschke) [inline lib/private/helper.php]: All hail to PHP type casting *sigh*

**Comment by LukasReschke** [comment]
Makes sense :+1:

**Comment by icewind1991** [comment]
:+1: looks good

File: tests/lib/helper.php [hunk 2]
```diff
@@ -53,21 +53,22 @@ public function phpFileSizeProvider()
 	}
 
 	/**
-	 * @dataProvider computerFileSizeProvider
+	 * @dataProvider providesComputerFileSize
```
Comment (PVince81) [inline tests/lib/helper.php]: You made the effort to rename it but kept the "computer" ? :wink:

**Pull request merged**